docs/source-pytorch/conf.py

```diff
@@ -325,6 +325,13 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     "graphcore": ("https://docs.graphcore.ai/en/latest/", None),
     "habana": ("https://lightning-ai.github.io/lightning-Habana/", None),
 }
+nitpicky = True
```

Comment:
I would land the current changes and disable this for now until we have resolved all of them :)

Reply:
```suggestion
nitpicky = False  # TODO: to be continued
```